src/client/clientlauncher.cpp

```diff
@@ -366,6 +366,12 @@ bool ClientLauncher::launch_game(std::string &error_message,
 	if (cmd_args.exists("password"))
 		menudata.password = cmd_args.get("password");
 
+
+	if (cmd_args.exists("password-file")) {
+		std::ifstream passfile(cmd_args.get("password-file"));
```

Comment:
needs a check on `passfile.good()` in case opening fails

Reply:
Just not doing anything doesn't seem like a good idea if opening fails, why not print an error message?